pyxem/generators/pdf_generator1d.py

```diff
@@ -33,10 +34,16 @@ class PDFGenerator1D:
     signal : ReducedIntensity1D
         A reduced intensity radial profile.
     """
-
+    @deprecated(since="0.15",
+                alternative="diffraction2d.get_pdf",
+                alternative_is_function=True,
+                removal="1.0.0", )
     def __init__(self, signal, *args, **kwargs):
         self.signal = signal
-
+    @deprecated(since="0.15",
```

Comment:
I suggest to have a blank line above the deprecation decorators (goes for more files).